### PR TITLE
Update Dashboard content from old PR.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

File: content/docs/configuration/dashboard.md [hunk 1]
```diff
@@ -5,10 +5,19 @@ weight: 6
 
 {{< callout type="info" >}} Available since 4.0.0-alpha.2.0 {{< /callout >}}
 
-The MKE Dashboard add-on provides the web UI to manage Kubernetes resources management:
+The MKE Dashboard add-on provides a web UI with which you can manage
```
Comment (brianbancroft) [inline content/docs/configuration/dashboard.md:8]: Re "with which you can manage". You're def the lead on docs, but I'm a fan of a more authoritative tone as it requires less words :)

**Review by brianbancroft — APPROVED** [review]
I have one small comment which you can ignore if you wish. Otherwise, is accurate and looks good to me.

File: content/docs/configuration/dashboard.md (hunk 1 above)
Comment (KoryKessel-Mirantis) [reply content/docs/configuration/dashboard.md]: Changed to "that you can use to manage". :-)

File: content/docs/configuration/dashboard.md [hunk 2]
```diff
@@ -5,10 +5,19 @@ weight: 6
...
 ![MKE dasboard preview](ui-preview.png)
 
-**To install the Dashboard add-on:**
+To access the MKE Dashboard, which is enabled by default, navigate to the
+address of the load balancer endpoint from a freshly-installed cluster. Refer
+to [Load balancer requirements](../../getting-started/system-requirements#load-balancer-requirements) for detailed information.
 
-The dashboard is enabled by default in the MKE installation.
+To disable the MKE Dashboard, set the `enabled` field to `false`
+in the `dashboard` section of the `config.yaml` file:
```
Comment (MagdaDziadosz) [inline content/docs/configuration/dashboard.md:18]:
```suggestion
in the `dashboard` section of the MKE configuration file:
```

**Review by MagdaDziadosz — CHANGES_REQUESTED** [review]

**Review by MagdaDziadosz — APPROVED** [review]

**Pull request merged**